cranelift/codegen/src/isa/x64/mod.rs

```diff
@@ -171,6 +171,24 @@ impl TargetIsa for X64Backend {
     fn function_alignment(&self) -> u32 {
         16
     }
+
+    #[cfg(feature = "disas")]
+    fn to_capstone(&self) -> Result<capstone::Capstone, capstone::Error> {
+        use capstone::prelude::*;
+        use target_lexicon::Architecture;
+
+        match self.triple().architecture {
+            Architecture::X86_32(_) => Capstone::new()
+                .x86()
+                .mode(arch::x86::ArchMode::Mode32)
+                .build(),
+            Architecture::X86_64 => Capstone::new()
+                .x86()
+                .mode(arch::x86::ArchMode::Mode64)
+                .build(),
```

Comment:
I'm wondering if you can minimize the filetest diffs by setting the disassembly syntax. I think it's defaulting to Intel syntax and we previously mostly followed AT&T syntax, right? Looks like you can call `builder.syntax(arch::x86::ArchSyntax::Att)` to request that.

Reply:
I like the switch back to AT&T, as it mirrors the MInst pretty-printing for x64.